libse/SubtitleFormats/SubRip.cs

```diff
@@ -180,7 +183,7 @@ private void ReadLine(Subtitle subtitle, string line, string next, string nextNe
 
                         if (_paragraph.Text.Length > 0)
                             _paragraph.Text += Environment.NewLine;
-                        _paragraph.Text += RemoveBadChars(line).TrimEnd().Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);
+                        _paragraph.Text += RemoveBadChars(line).Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);

```

Comment:
Tadaa 😄!!! I found something: calling **Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);** is redundant, line is element of "text.SplitToLines()"; right?

I'll send patch later, so it doesn't conflict with this one.